docs/_includes/index_navbar.html

```diff
@@ -0,0 +1,12 @@
+<ul class="list-group">
+    <a class="list-group-item list-group-item-action flex-column align-items-start active bg-blue">
```

Comment:
I would prefer another color for the "Resources" header box. From the used blue I would infer that the box is interactive/can be clicked or is a currently selected tab.

Reply:
Do you have any idea for alternatives? I create two other versions:

**CURRENT VERSION**
![index](https://user-images.githubusercontent.com/7346610/43840982-5f6777f6-9b22-11e8-838d-760971bfbb19.png)

**ALTERNATIVE VERSION 1**
* Resources entry has the same color as the navigation bar of the other sites
![res1](https://user-images.githubusercontent.com/7346610/43946479-ab5cc792-9c85-11e8-9dbb-81b709cf17d5.png)

**ALTERNATIVE VERSION 2**
* Removed the resources entry. Used light blue for list entries and made jumbotron white
* If hover over a list entry, the background of the entry becomes white
![res2](https://user-images.githubusercontent.com/7346610/43946489-b1877a2c-9c85-11e8-8882-85638cc49854.png)

Reply:
Both look fine to me. I would prefer alternative 2, but that is just a matter of personal preference.

Reply:
I like the second one more, too.

Reply:
Implemented "Alternative 2"